includes/Util.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace NewfoldLabs\WP\Module\HelpCenter;
+
+/**
+ * The utility pass through for interacting with the
+ * AI service
+ */
+class Util {
+	/**
+	 * The function to proxy to the AI service and get a response
+	 *
+	 * @param integer $post_id the Post Id
+	 * @param string  $status  the status
+	 */
+	public static function post_status (
```

Comment:
`post_status` is a bit too generic. Can we name it something like `post_feedback` or `add_feedback`